no-computer.md

```diff
@@ -11,41 +11,74 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+I believe Lego can be a DSL. The need for Lego is met by people's desires to
+build something of their own design with lego blocks. This can be thought of 
+as running a program (the end product) with pre-defined functions (the lego 
+blcoks).
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Lego is very domain-specific, in a way that it let the users use their 
+creativity to build something with ease since the lego blocks are 
+pre-manufactured. 
+So the user does not have to focus on the details of small pieces of their 
+big picture design.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+As a kid, I liked building different things with Lego blocks.
+
+Some Lego packages come with a picture of specific designs, such as 
+Architecture - the White House. With those, you know from the beginning
+what your end product SHOULD look like. However, I find it fun to
+build my own thing and buy different Lego blocks along the way as needed.
```

Comment:
Yay! I too enjoyed building without the instructions and seeing where I could end up.